examples/visitors/go.sum

```diff
@@ -0,0 +1 @@
+github.com/bitfield/script v0.9.0/go.mod h1:MRqXqj+GhENKHXcjadvDTU5/Z1rgBHbtw/RSyHprCl4=
```

Comment:
We probably don't want to check in the `go.sum` for the example programs.

Reply:
This is the second time I made the mistake, do you mind if I put it in the gitignore?

Reply:
Go ahead! That's a good idea.